File: areatriangulo.java
import java.util.Scanner;

class areatriangulo{
static Scanner teclado=new Scanner(System.in);
public static void main(String [] arg){
  //Definir variables y otros
  System.out.println("Hola Mundo");
  int b, h, area;
  //Datos de entrada
  System.out.println("Ingrese la base: ");
  b=teclado.nextInt();
  System.out.println("Ingrese la altura: ");
  h=teclado.nextInt();
  //Proceso
  area=(b*h)/2;
  //Datos de salida
  System.out.println("Área de un triángulo es : "+area);
}
}

File: distinct-language/areatriangulo.go
package main
import "fmt"
func main() {
//Declaracion de variables y otros
fmt.Println("Ejercicio 01: Área Triángulo")
var b int
var h int
var area float64
//Datos de lectura
fmt.Println("Ingrese Base:")
fmt.Scanln(&b)
fmt.Println("Ingrese Altura:")
fmt.Scanln(&h)
//Proceso
area=float64((b*h)/2)
//Datos de salida
fmt.Println("El área es: ", area)
}